### DISCUSSION TITLE
Stop ratings from being overwritten

### DISCUSSION THREAD

**Discussion by RileyXX** [Taxel/PlexTraktSync#1485] [General]
Are there any config settings to stop ratings from being overwritten? Sometimes I change a rating later on in Trakt and then it is overwritten with the rating from Plex. I would prefer no ratings be overwritten that way I can change ratings in Plex or Trakt without worrying about them being changed later on.

**Comment by simonc56** [comment]
Yes you can edit config.yml to disable ratings sync.

**Reply by RileyXX** [reply]
Thanks for the reply, I still want ratings to sync just don't want them to be overwritten if they're already set.

**Reply by simonc56** [reply]
Not possible. This is a sync program, so it cannot let things unsynced after a sync.

**Reply by RileyXX** [reply]
I think when syncing items it should check if there is already a rating for that item otherwise skip setting the rating for that item. Because if I change my rating on Trakt afterwards and run the sync again my new rating will be overwritten. I have other programs that are syncing my ratings from Trakt to IMDB etc so I like to keep my ratings in Trakt updated, but PlexTraktSync keeps overwriting my Trakt ratings with outdated ratings from Plex. Hope that makes sense.

**Reply by simonc56** [reply]
Ok you can [create](https://github.com/Taxel/PlexTraktSync/issues/new/choose) a Feature request.

**Reply by RileyXX** [reply]
Will do thanks!

**Comment by RileyXX** [comment]
Opened up a new issue here https://github.com/Taxel/PlexTraktSync/issues/1489

And a new pull request to fix this issue here https://github.com/Taxel/PlexTraktSync/pull/1496

**Comment by glensc** [comment]
Closing in favor of feature request
